aioxmpp/bookmarks/service.py

```diff
@@ -290,58 +303,107 @@ def sync(self):
         yield from self.get_bookmarks()
 
     @asyncio.coroutine
-    def add_bookmark(self, new_bookmark):
+    def add_bookmark(self, new_bookmark, max_retries=3):
```

Comment:
Please add a description of the ``max_retries`` argument here (with ``:param max_retries: …\n:type max_retries: :class:`int` ``) and also make ``max_retries`` kwarg-only.